artifacts/testing/C01-device-errors.feature

```diff
@@ -0,0 +1,118 @@
+Feature: CAMARA Common Artifact C01 - Test scenarios for device errors
+
+    Common error scenarios for POST operations with device as input either in the request
+    body or implied from the access.
+
+    Artifact parameters (to be replaced by values according to the API operation):
+    
+    - {{feature_identifier}} 
+
+    This is not a complete feature but a collection of scenarios that can be applied
+    with minor modifications to test plans.
+
+    These scenarios assume that other properties not explicitly mentioned in the scenario
+    are set by default to a valid value. This can be specified in the feature Background.
+
+    # Error scenarios for management of input parameter device
+
+    # If the access token identifies a device, error 422 UNNECESSARY_DEVICE may be returned instead

```

Comment:
Scenario details are not clear,if we expect 422 then we anticipate 422 being considered and written in below mentioned scenario where we see only 400.

Reply:
I have the misleading comment, here it should be 400